lib/core/language-menu/language-picker.component.stories.ts

```diff
@@ -0,0 +1,61 @@
+import { LanguagePickerComponent } from './language-picker.component';
+import { LanguageMenuComponent } from './language-menu.component';
+import { LanguageMenuModule } from './language-menu.module';
+import { MatMenuModule } from '@angular/material/menu';
+import { CoreStoryModule } from './../testing/core.story.module';
+import { componentWrapperDecorator, Meta, moduleMetadata, Story } from '@storybook/angular';
+import { MatIconModule } from '@angular/material/icon';
+import { TranslateModule } from '@ngx-translate/core';
+
+export default {
+    component: LanguagePickerComponent,
+    title: 'Core/Components/Language Menu/Language Picker',
+    decorators: [
+        moduleMetadata({
+            imports: [CoreStoryModule, LanguageMenuModule, MatMenuModule, MatIconModule, TranslateModule]
+        })
+    ]
+} as Meta;
+
+const Template: Story<LanguageMenuComponent> = (args) => ({
+    props: args
+});
+
+export const Default = Template.bind({});
+Default.decorators = [
+    componentWrapperDecorator(story =>`
+      <style>
+        table {font-family: arial, sans-serif;border-collapse: collapse;margin-top:10px;}
+        td, th {border: 1px solid #dddddd;text-align: left;padding: 8px;}
+      </style>
+      ${story}
+      <table>
+        <tr><th>Example key</th><th>Value</th></tr>
+        <tr><td>CORE.HOST_SETTINGS.TITLE</td><td>{{ 'CORE.HOST_SETTINGS.TITLE' | translate }}</td></tr>
+        <tr><td>CORE.METADATA.ACTIONS.SAVE</td><td>{{ 'CORE.METADATA.ACTIONS.SAVE' | translate }}</td></tr>
+        <tr><td>ADF.LANGUAGE</td><td>{{ 'ADF.LANGUAGE' | translate }}</td></tr>
+      </table>
+    `)
+];
+
+export const AsNestedMenu = Template.bind({});
+AsNestedMenu.decorators = [
+    componentWrapperDecorator(story => `
+      <style>
```

Comment:
I am not sure about this custom style, all the styles should come from the app itself, what do we do in the others stories?

Reply:
Removed in new push.